pkg/kubectl/cmd/autoscale.go

```diff
@@ -54,7 +54,7 @@ func NewCmdAutoscale(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 	}
 	cmdutil.AddPrinterFlags(cmd)
-	cmd.Flags().String("generator", "horizontalpodautoscaler/v1beta1", "The name of the API generator to use. Currently there is only 1 generator.")
```

Comment:
Fixed and commented https://github.com/kubernetes/kubernetes/pull/20501#discussion_r51868268